packages/eslint-config-airbnb-base/rules/errors.js

```diff
@@ -74,6 +74,10 @@ module.exports = {
     // http://eslint.org/docs/rules/no-prototype-builtins
     'no-prototype-builtins': 2,
 
+    // disallow use of unary operators, ++ and --
+    // http://eslint.org/docs/rules/no-plusplus
+    'no-plusplus': 'error',
```

Comment:
Also, according to http://eslint.org/docs/rules/#stylistic-issues this rule is in "style", so please delete the relevant lines from this file, and move them to lines 183-185 in `rules/style.js`